contracts/random/offchainsources/IOffchainRandomSource.sol

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Immutable Pty Ltd 2018 - 2023
+// SPDX-License-Identifier: Apache 2
+pragma solidity 0.8.19;
+
+/**
+ * @notice Off-chain random source adaptors must implement this interface.
+ */
+interface IOffchainRandomSource {
+    // The random seed value is not yet available.
+    error WaitForRandom();
+
+    /**
+     * @notice Request that an offchain random value be generated.
+     * @return _fulfilmentIndex Value to be used to fetch the random value in getOffchainRandom.
+     */
+    function requestOffchainRandom() external returns (uint256 _fulfilmentIndex);
```

Comment:
nit but inconsistency between american vs aus spelling of `fulfilment` (i.e. some uses of `fulfilment` vs `fulfillment`

Reply:
I had fixed that - but obviously not everywhere / had added more fulfilllllllments !

Reply:
Pushed changes to make consistent